i18n/messages/en-US.json

```
@@ -202,6 +383,356 @@
+        "exercise": "Exercise",
+        "video": "Video"
+      },
+      "actions": {
+        "prescribe": "Prescribe",
+        "edit": "Edit",
+        "delete": "Delete",
+        "preview": "Preview"
+      }
+    }
+  },
+  "privacy": {
```

Biome (2.1.2)

[error] The key privacy was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)